internal/fields/validate.go

```diff
@@ -153,6 +154,14 @@ type Validator struct {
 	disabledNormalization bool
 
 	injectFieldsOptions InjectFieldsOptions
+
+	esAPI *elasticsearch.API
+
+	indexTemplateName string
+
+	dataStreamName string
+
+	LocalSchema []FieldDefinition
```

Comment:
New variable to hold just the Field Definitions from the local directory (package).

It is needed to check if a given mapping is related to a field definition with type `array`.